compiler+runtime/src/cpp/jank/codegen/processor.cpp

```diff
@@ -186,6 +194,12 @@ namespace jank::codegen
                              typed_o->name);
             }
           }
+          else if constexpr(std::same_as<T, runtime::obj::character>)
+          {
+            fmt::format_to(inserter,
+                           R"(jank::make_box<jank::runtime::obj::character>({}))",
```

Comment:
I don't think this will work. Have you tried doing something like `(fn [] \space)` in a repl?